### ISSUE TITLE
WARNING: using ModelingToolkit.Reaction in module SBML conflicts with an existing identifier.

### ISSUE THREAD

**Issue by anandijain** [LCSB-BioCore/SBML.jl#48]
```julia
julia> using ModelingToolkit, SBML
[ Info: Precompiling ModelingToolkit [961ee093-0014-501f-94e3-6117800e7a78]
[ Info: Precompiling SBML [e5567a89-2604-4b09-9718-f5f78e97c3bb]
WARNING: using ModelingToolkit.Reaction in module SBML conflicts with an existing identifier.
```

Do we want to resolve this? I suppose neither package wants to rename their `Reaction` struct 😛.

**Comment by exaexa** [comment]
I was already thinking about internalizing all names to become `SBML.Reaction` etc, because we had the same conflict already :] So yeah, let's unexport Reaction, Species, and others. PR welcome (I'll get to it on friday earliest :D).

**Comment by anandijain** [comment]
closed by #50
